frontend-shared/README.md

```diff
@@ -0,0 +1,26 @@
+# Hypothesis frontend shared
```

Comment:
I think a more human-readable name/title here might be helpful: "Shared resources for Hypothesis front-end applications" or similar